state-sync/src/coordinator.rs

```diff
@@ -1128,16 +1140,23 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
             .execute_chunk(txn_list_with_proof, target, intermediate_end_of_epoch_li)
     }
 
+    /// Returns true if consensus is currently executing and state sync should
+    /// therefore not write to storage. Reads are still permitted (e.g., to
+    /// handle chunk requests).
+    fn is_consensus_executing(&mut self) -> bool {
+        self.is_initialized() && self.role == RoleType::Validator && self.sync_request.is_none()
```

Comment:
Why does this mean "consensus is running"? I thought it means the syncing is started?